symcache/tests/snapshots/test_writer__functions_linux.snap

```diff
@@ -478,13 +471,9 @@ expression: FunctionsDebug(&symcache)
             487b DumpOSInformation
             488d LogAppend
             48cc LogAppend
-            48e4 LogAppend
-            48fc LogAppend
-            4914 LogAppend
```

Comment:
All functions that are removed from the snapshots did not contain any line records.